modules/460-log-shipper/hooks/internal/vector/transform/extra_fields.go

```diff
@@ -112,3 +60,75 @@ func ExtraFieldTransform(extraFields map[string]string) *DynamicTransform {
 
 	return &extraFieldsTransform
 }
+
+func processExtraFieldKey(key string, value string) string {
+	key = escapeVectorString(key)
+	if validMustacheTemplate.MatchString(value) {
+		dataField := validMustacheTemplate.FindStringSubmatch(value)[1]
+		if dataField == "parsed_data" {
```

Comment:
Looks like kind of an odd hack that is not documented. Could you please at least highlight it with the comment?